src/qt/locale/bitcoin_ja.ts

```diff
@@ -2736,7 +2736,7 @@ Note:  Since the fee is calculated on a per-byte basis, a fee of "100 tapyrus pe
         <translation>トランザクションの全体サイズ</translation>
     </message>
     <message>
-        <source>Transaction virtual size</source>
+        <source>Transaction size</source>
         <translation>トランザクションの仮想サイズ</translation>
```

Comment:
```suggestion
        <translation>トランザクションサイズ</translation>
```

And other languages as well.

Reply:
Done! I didn't think of google translate before!

Reply:
Can you replace other `仮想サイズ` to `サイズ` as well.